mod.conf

```diff
@@ -0,0 +1,2 @@
+name = frameglass
```

Comment:
framedglass*

Reply:
oh, wow, thanks for spotting that, completely missed that :+1:  @Panquesito7 do you have time for this? I could open another PR with the changes if you want...